package.json

```diff
@@ -378,6 +378,7 @@
     "@metamask/eslint-config-typescript": "^9.0.1",
     "@metamask/forwarder": "^1.1.0",
     "@metamask/phishing-warning": "^2.1.0",
+    "@metamask/safe-event-emitter": "^2.0.0",
```

Comment:
The older `safe-event-emitter` package is in `dependencies` still. We should be able to update that rather than adding another copy. I've done that in a separate PR: https://github.com/MetaMask/metamask-extension/pull/18419